test/performance/README.md

```diff
@@ -15,32 +15,101 @@ different rate, you can write your own pacer by implementing
 interface. Custom pacer implementations used in Knative tests are under
 [pacers](https://github.com/knative/pkg/tree/main/test/vegeta/pacers).
 
-## Benchmarking using Mako
 
-The benchmarks were originally built to use [mako](https://github.com/google/mako), but currently
-running without connecting to the Mako backend, and collecting the data using
-a Mako sidecar stub.
+## Testing architecture
 
-### Run without Mako
+The performance tests are based on Kubernetes Jobs running Golang code based on different [benchmarks](#benchmarks).
+The script [performance-tests.sh](./performance-tests.sh) first creates a cluster in GKE, installs Serving with specific settings
+for the performance tests. Then it installs the required Knative Services and runs the testing jobs.
 
-To run a benchmark once, and use the result from `mako-stub` for plotting:
+The results are written to:
+* Stdout
+* A logfile in a folder defined in env: `$ARTIFACTS`
+* To an InfluxDB hosted in the knative-community GKE project: https://github.com/knative/infra/tree/main/infra/k8s/shared
```

Comment:
Yeah I assumed that as well. I first tried to do the dashboards in influx, but that is very limited and also pretty slow if you add multiple graphs. So I'm in favour of

> Probably worth doing this in a follow up

/cc @skonto. Maybe we could use this downstream as well (instead of ElasticSearch directly?)